server/apps/users/models.py

```diff
@@ -27,8 +27,8 @@ class Todo(TimestampedModel, AuditModel):
     description = models.TextField(blank=True, null=True)
     date = models.DateTimeField(default=now)
     is_bookmarked = models.BooleanField(default=False)
-    owner = models.ForeignKey(User, related_name='set_owner', on_delete=models.CASCADE)
```

Comment:
owner also, cannot be blank or null

Reply:
I have removed the null constrains and the default value for `owner` is set to 'anonymous'. This is because when we pass the request data to serializer then if the owner field is null the serializer validation fails. To make the serializer validation pass, the default value of owner is set the 'anonymous' and in the `create` method of the TodoSerializer I'm setting the actual value of the owner.